myapp/linearRegression.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+file_path = '../data/Course_Summary_2022_2023.json'
+df = pd.read_json(file_path)
+print(df.head())
+from sklearn.model_selection import train_test_split
+
+X = df['Cap'].values.reshape(-1, 1)

```

Comment:
Then make a method for the regression algorithm and call it from the main method